tests/e2e/specs/infobox.spec.js

```diff
@@ -0,0 +1,80 @@
+/// <reference types="cypress" />
+
+import { forEachTestViewport } from '../support'
+import features from '../fixtures/features.fixture.json'
+
+function longClickOnMap() {
+    cy.readWindowValue('map').then((map) => {
+        cy.simulateEvent(map, 'pointerdown')
+        cy.wait(500) // eslint-disable-line
+        cy.simulateEvent(map, 'pointerup')
+        cy.simulateEvent(map, 'singleclick')
+    })
+}
+
+describe('The infobox', () => {
+    forEachTestViewport((viewport, isMobile, isTablet, dimensions) => {
+        context(
+            `viewport: ${viewport}`,
+            {
+                viewportWidth: dimensions.width,
+                viewportHeight: dimensions.height,
+            },
+            () => {
+                beforeEach(() => {
+                    const layer = 'test.wmts.layer'
+                    cy.intercept('**/MapServer/identify**', features)
```

Comment:
Sorry to come back again, but could you also change how this fixture is imported/used 😖

Reply:
The thing here is that I need to import the features anyway because I need to only send one feature for the intercept on line 30.

Reply:
Then you need to access the fixture in a `cy.fixture` method.
see https://docs.cypress.io/api/commands/fixture#Accessing-Fixture-Data
and an actual exemple in our code here : https://github.com/geoadmin/web-mapviewer/blob/ea5adc10dc74ac1ae694587d28784e3d90d199a3/tests/e2e/specs/layers.spec.js#L115-L130

Reply:
Ah nice :sunglasses: Thank you!